screens/signup/notifications.js

```diff
@@ -0,0 +1,26 @@
+// import { Notifications } from 'expo';
```

Comment:
@anniero98 is this file being used in the project right now?

Reply:
No it is not! Will remove